plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+            "isIntraSubnetRoutingEnabled": anycast_details_in_ccc.get("isIntraSubnetRoutingEnabled")
+        }
+        params_in_playbook = ["tcp_mss_adjustment", "traffic_type", "security_group_name", "layer2_flooding_enabled",
+                              "fabric_enabled_wireless", "ip_directed_broadcast", "multiple_ip_to_mac_addresses",
+                              "supplicant_based_extended_node_onboarding", "group_policy_enforcement_enabled"]
+
+        anycast_mapping = self.get_anycast_gateway_mapping(vn_name)
+
+        if vn_name == "INFRA_VN":
+            params_to_remove = ["security_group_name", "layer2_flooding_enabled", "fabric_enabled_wireless",
+                                "ip_directed_broadcast", "multiple_ip_to_mac_addresses"]
+            for param in params_to_remove:
+                params_in_playbook.remove(param)
+            anycast_payload.pop("isCriticalPool", None)
+            anycast_payload.pop("isIntraSubnetRoutingEnabled", None)
+        else:
+            params_in_playbook.remove("supplicant_based_extended_node_onboarding")
+            params_in_playbook.remove("group_policy_enforcement_enabled")
+            anycast_payload.pop("poolType", None)
+
+        if anycast.get("traffic_type") and anycast_details_in_ccc.get("isCriticalPool") is True:
+            params_in_playbook.remove("traffic_type")
+            anycast_payload["trafficType"] = anycast_details_in_ccc.get("trafficType")
+
+        for param in params_in_playbook:
+            key = anycast_mapping.get(param)
+            if anycast.get(param) is not None:
+                anycast_payload[key] = anycast.get(param)
+            else:
+                anycast_payload[key] = anycast_details_in_ccc.get(key)
+
+        return anycast_payload
+
+    def add_anycast_gateways_in_system(self, collected_add_anycast_payload):
+        """
+        Adds Anycast Gateways to the Cisco Catalyst Center using the provided payload.
+        Args:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            collected_add_anycast_payload (dict): A dictionary containing the necessary details for
+                adding Anycast Gateways.
+        Returns:
+            self (object): The instance of the class, allowing for method chaining. The status of the operation
+                can be checked via the `status` attribute.
+        Description:
+            This function interacts with the Cisco DNA Center API to add Anycast Gateways. It sends the provided
+            payload to the API and processes the response.
+            In case of an exception during the API call, the function captures the exception and logs the error.
+            The method returns the instance itself, allowing for further interactions with the object.
+        """
+
+        try:
+            payload = {"payload": collected_add_anycast_payload}
+            task_name = "add_anycast_gateways"
```

Comment:
self.log("Constructing API call payload for task '{0}': {1}".format(task_name, payload), "DEBUG")